Drift/Private/DriftFlexmatch.cpp

```diff
@@ -143,27 +188,38 @@ void FDriftFlexmatch::StopMatchmaking()
 	{
 		return;
 	}
-	auto Request = RequestManager->Delete(FlexmatchURL);
+	if ( CurrentTicketUrl.IsEmpty() )
+	{
+		UE_LOG(LogDriftMatchmaking, Warning, TEXT("FDriftFlexmatch::StopMatchmaking - Cancelling without a known ticket"));
+		return;
+	}
+	auto Request = RequestManager->Delete(CurrentTicketUrl);
 	Request->OnError.BindLambda([this](ResponseContext& context)
 	{
 		UE_LOG(LogDriftMatchmaking, Error, TEXT("FDriftFlexmatch::StopMatchmaking - Failed to cancel matchmaking"
 					", Response code %d, error: '%s'"), context.responseCode, *context.error);
 	});
 	Request->OnResponse.BindLambda([this](ResponseContext& context, JsonDocument& doc)
 	{
+		FDriftFlexmatchTicketDeleteResponse Response;
+		if (!JsonArchive::LoadObject(doc, Response))
+		{
+			UE_LOG(LogDriftMatchmaking, Error, TEXT("FDriftFlexmatch::StopMatchmaking - Failed to parse response from DELETE to %s"
+						", Response code %d, error: '%s'"), *FlexmatchTicketsURL, context.responseCode, *context.error);
```

Comment:
Do you mean to return here, or continue without a valid response?

Reply:
that's a booboo; needs fixing.  Also the StatusString below should be gone...